gapic/src/main/com/google/gapid/image/Image.java

```diff
@@ -80,6 +80,11 @@
    */
   public boolean isHDR();
 
+  /**
+   * @return true if this image should be tone-mapped when displayed.
+   */
+  public boolean isCount();
```

Comment:
Combine `isHDR` and `isCount` into an enum, since there really are only 3 possibilities:
LDR, HDR and Count.

Reply:
Done